boards/nxp/frdm_mcxw71/frdm_mcxw71.dts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2024 NXP
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nxp/nxp_mcxw71.dtsi>
+#include "frdm_mcxw71-pinctrl.dtsi"
+
+/ {
+	model = "NXP FRDM-MCXW71 board";
+
+	aliases {
+		led0 = &green_led;
+	};
+
+	chosen {
+		zephyr,flash = &flash;
+		zephyr,flash-controller = &fmu;
+		zephyr,code-partition = &code_partition;
+		zephyr,sram = &stcm0;
+		zephyr,console = &lpuart1;
+		zephyr,shell-uart = &lpuart1;
+	};
+
+	user_led {
+		compatible = "gpio-leds";
+		green_led: led {
+			gpios = <&gpioc 1 GPIO_ACTIVE_LOW>;
+		};
+	};
+};
+
+&gpioc {
+	status = "okay";
+};
+
+&lpuart1 {
+	current-speed = <115200>;
+	status =  "okay";
+	pinctrl-0 = <&pinmux_lpuart1>;
+	pinctrl-names = "default";
+};
+
+&flash {
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		code_partition: partition@0 {
```

Comment:
Based on the partition setup it looks like we aren't enabling MCUBoot here. Is there a reason for that? Usually we can just check it off when we enable the flash driver. It looks like we have a small enough write size MCUBoot *should* just work

Reply:
it's not for mcu boot, BLE will use the storage partition

Reply:
Ah ok, so we'll enable MCUBoot in a later PR. No worries then.

Reply:
yes the mcu boot is tracked separately in internal tickets